buildlog/buildlog.go

```diff
@@ -57,39 +59,49 @@ func (l log) Stop() {
 
 func (l log) Run() {
 	reader := bufio.NewReader(l.file)
+	buildDone := false
 
 	for {
-		select {
-		case <-l.ctx.Done():
-			break
-		default:
-			err := l.output(reader)
-			if err != nil {
-				logrus.Errorf("failed to run logger: %v\n", err)
-				logrus.Info("But build is not stopping")
-				l.cancel()
+		if !buildDone {
+			select {
+			case <-l.ctx.Done():
+				buildDone = true
+			default:
 			}
 		}
+
+		readDone, err := l.output(reader)
+		if err != nil {
+			logrus.Errorf("failed to run logger: %v\n", err)
+			logrus.Info("But build is not stopping")
```

Comment:
How about using positive words?
```suggestion
			logrus.Info("But build is still running")
```

Reply:
Thx. I fixed it.